packages/widget/src/hooks/useChains.ts

```diff
@@ -17,34 +17,36 @@ export const useChains = () => {
       staleTime: 300000,
     });
 
-  const { data: filteredChains, isLoading: isLoadingFilteredChains } = useQuery(
-    ['filtered-chains', availableChains?.length, keyPrefix],
-    async () => {
-      if (!availableChains) {
-        return;
-      }
-      const filteredChains = availableChains.filter((chain) =>
-        isItemAllowed(chain.id, chains),
-      );
-      const chainOrder = initializeChains(
-        filteredChains.map((chain) => chain.id),
-      );
-      const [fromChainValue, toChainValue] = getValues([
-        FormKey.FromChain,
-        FormKey.ToChain,
-      ]);
-      if (!fromChainValue) {
-        setValue(FormKey.FromChain, chainOrder[0]);
-      }
-      if (!toChainValue) {
-        setValue(FormKey.ToChain, chainOrder[0]);
-      }
-      return filteredChains;
-    },
-    {
-      enabled: Boolean(availableChains),
-    },
-  );
+  const filteredChains = useMemo(() => {
```

Comment:
I've switch this to useMemo which appears to work. Would be good to have a conversation just to check if there was a reason that useQuery was being used previous for a specific reason or not.

Reply:
We are using the useQuery hook to provide caching across the app so we don't have to keep regenerating the list. Will move this back tot he previous commit